index.html

```diff
@@ -41,11 +41,63 @@
         background: white;
         padding: 0.5em;
       }
+      .translateme::before {
+        content: ' Translate this: ';
+        color: coral;
+        border-left: 4px solid coral;
+        padding-left: .5em;
+      }
+      .translateme {
+        background-color: #F3F3C3;
+      }
+      .checkme {
+        background-color: #BBEFFC;
+      }
+      .checkme::before {
+        content: ' Check this translation: ';
+        color: coral;
+        border-left: 4px solid coral;
+        padding-left: .5em;
+      }
     </style>
   </head>
 <body>
 <section id="abstract">
-  <p>A simple set of rules for placement of Ruby text in Japanese typography.</p>
+  <p class="checkme">This document describes simple method of processing for Ruby 
+  in Japanese layout realized with technologies like CSS, SVG, and XML-FO, 
+  as references to rendering engine implementors. 
```

Comment:
> This document describes simple method of processing for Ruby in Japanese layout realized with technologies like CSS, SVG, and XML-FO, as references to rendering engine implementors.

Maybe change it to something like:

> This document describes a simple method of ruby composition for Japanese layout realized with technologies like CSS, SVG, and XML-FO, as basic information for implementations.

Reply:
how about with updated?

Reply:
I would use lowercase for "ruby", since it is lowercase in [HTML](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-ruby-element), [CSS](https://www.w3.org/TR/css-ruby-1/), and many other places like the [What is ruby?](https://www.w3.org/International/questions/qa-ruby) i18n article.

Reply:
done.